vi/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: news_post
+title: "Ruby 2.3.0-preview2 Released"
+author: "naruse"
+translator: "Trung Lê"
+date: 2015-12-11 14:00:00 +0000
+lang: vi
+---
+
+Chúng tôi hân hạnh công bố ấn bản Ruby 2.3.0-preview2.
+
+Ruby 2.3.0-preview2 là phiên bản thử nghiệm thứ hai của Ruby 2.3.0.
+Giới thiệu cùng với phiên bản này bao gồm các cải tiến sau:
+
+[Frozen String Literal
+Pragma](https://bugs.ruby-lang.org/issues/11473) được giới thiệu. Trên
+Ruby 2.1, "str".freeze được tối ưu hoá để giảm số lượng đối tượng được
+tạo ra. Ruby 2.3 cho phép đóng băng tất cả chuỗi ký tự trong các
+file mã nguồn thông qua một magic comment và một cờ của command line.
+Ngoài ra người dùng còn có thể biết được vị trí đổi tượng được tạo
+khi gặp phải lỗi "can't modify frozen String" thông qua cờ `--enable-frozen-string-literal-debug`.
+
+[Safe navigation operator](https://bugs.ruby-lang.org/issues/11537)
+([hay còn gọi là lonely operator](https://instagram.com/p/-M9l6mRPLR/)) `&.`,
+chức năng này đã hiện diện trong C#, Groovy, và Swift. Cú pháp này
+được giới thiệu để làm giảm phiền toái xử lý `nil`, xem vd:
+`obj&.foo`. `Array#dig` và `Hash#dig` cũng mới được bổ sung vào thư viện.
+
+[did_you_mean gem được kèm cùng](https://bugs.ruby-lang.org/issues/11252).
```

Comment:
`did_you_mean gem được kèm cùng` is not clear, but I don't know which is better :smile: